scripts/dist.ts

```diff
@@ -137,6 +146,32 @@ function toDist(sourcePath: string): YAML.Document {
   return yaml;
 }
 
+function convertMarkdown(yaml: YAML.Document) {
+  // Markdown is only used the descriptions right now, but could easily be
+  // supported in names as well.
+  const description = yaml.get("description");
+  if (typeof description !== "string") {
+    return;
+  }
+
+  const text = remark()
```

Comment:
I'm not happy with using both `remark()` and `unified()` and parsing the Markdown twice. There must be a way to parse the Markdown string once, and turn the resulting document into both plaintext and HTML.

Reply:
Sorry, I think I might've steered you wrong with my previous comment. I think something like this would work well (enough for now):

(hast-util-to-string does the same thing as the DOM's `textContent`)

```typescript
  import { toString as hastTreeToString } from "hast-util-to-string";
  import rehypeStringify from "rehype-stringify";
  import remarkParse from "remark-parse";
  import remarkRehype from "remark-rehype";
  import { unified } from "unified";

  // snip
  
  const mdTree = unified().use(remarkParse).parse(description);
  const htmlTree = unified().use(remarkRehype).runSync(mdTree);
  const text = hastTreeToString(htmlTree);
  let html = unified().use(rehypeStringify).stringify(htmlTree);
```

Reply:
Getting the plaintext via HTML instead of directly from the Markdown tree adds a step between Markdown and plaintext that shouldn't be strictly needed, but in terms of dependencies and lines of code it's no worse. I'll try it.